kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSValueArgumentImpl.kt

```diff
@@ -51,4 +51,8 @@ class KSValueArgumentImpl private constructor(
     override fun <D, R> accept(visitor: KSVisitor<D, R>, data: D): R {
         return visitor.visitValueArgument(this, data)
     }
+
+    override fun toString(): String {
+        return "${name?.asString() ?: ""}:$value"
```

Comment:
If `name` is `null`, we'll get something like `:v`?

Also, if it is a named argument, I expected "name = value" instead.

Reply:
yes we will get something like you said. the `toString()` method currently tries to align with the compiler plugin implementation, might change it later if this is better representation.